integrationTests/vm/mockVM/vmGet/vmGet_test.go

```diff
@@ -14,7 +14,9 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process"
 	"github.com/ElrondNetwork/elrond-go/process/factory"
 	"github.com/ElrondNetwork/elrond-go/process/smartContract"
+	"github.com/ElrondNetwork/elrond-go/process/sync/disabled"
 	"github.com/ElrondNetwork/elrond-go/state"
+	state2 "github.com/ElrondNetwork/elrond-go/testscommon/state"
```

Comment:
state2

Reply:
no need after removing accounts from the struct :smile: